index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<p class="advisement">You probably need to store the name in both Latin and native scripts, in which case you will need to ask the user to submit their name in both native script and Latin-only form, as separate items.</p>
+	<details class="links"><summary>source</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#encoding">Implications for character support</a>, in <cite>Personal Names Around the World</cite>.</p>
+	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
+	</div>
+
+<p>The need for multiple fields will depend to some extent on what you are collecting people’s names for, and how you intend to use them.</p>
+<ul>
+<li>Are you collecting the person’s name just to have an identifier in your system?  If so, it may not matter whether the name is stored in ASCII-only or native script.</li>
+
+<li>Or will they be called by name on a welcome page or in correspondence?  If you will correspond using their name on pages written in their language, it would seem sensible to have the name in the native script.</li>
+
+<li>Is it important for people in the organization that handles queries to be able to recognise and use the person’s name?  If so, you may need a Latin transcription.</li>
+
+<li>Will their name be displayed or searchable (for example Flickr optionally shows people’s names as well as their user name on their profile page)?  Or will you want to send them correspondence in their own language, but track them in your back-office in a language such as English? If so, it may be necessary to store the name in both Latin and native scripts, in which case you probably need to ask the user to submit their name in both native script and Latin-only form, using separate fields.</li>
+</ul>
+
+
+	<div class="req" id="loc_names_transcription">
+	<p class="advisement">Provide a field for a transcription of the name, where necessary.</p>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#encoding">Implications for character support</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>For example, Japanese users may need to provide a transcription in a Japanese syllabic script rather than/in addition to the ideographic form. This field is used for sorting Japanese names, but also allows someone looking at the name to check how it is pronounced.</p>
```

Comment:
We should probably add to the sorting section... at a later date.

Reply:
tbd